### DISCUSSION TITLE
Pushover Info Messages

### DISCUSSION THREAD

**Discussion by kalleick** [dewenni/ESP_Buderus_KM271#92] [Q&A] [answered]
Ich bin Fan von den Pushover Nachrichten, vielen Dank nochmals.

Dazu zwei Fragen: 
a) es liegt vermutlich an der Logamatic 2107 selbst, dass ich nur Nachrichten bekomme, wenn das System in den Frostmodus wechselt, es gibt keine Nachrichten, dass der Frostmodus verlassen wird, oder?
![IMG_7094](https://github.com/dewenni/ESP_Buderus_KM271/assets/95974901/9b3211ff-0dd0-4c6b-9239-2b47b23339aa)
b) wenn ich einen Wert ändere, z.B. die Warmwasser Temperatur von 56 auf 58 Grad, dann bekomme ich auch eine Info Nachricht: Message: setvalue: ww_soll - empfangen. Wäre es hier möglich den neuen Zielwert (hier 58) ebenfalls mit in die Pushover Nachricht mit aufzunehmen oder kommt der Wert nicht mit rüber?

Vielen Dank
Patrick

**Comment by dewenni** [comment, marked as answer]
Hi @kalleick,
danke für dein Feedback. Ich habe beide Anmerkungen implementiert und werde es mit dem nächsten Release veröffentlichen.

Generell gibt es aktuell folgende Nachrichten von der Logamatik:
```
Heizung hat in Sommerbetrieb gewechselt ☀️
Heizung hat in Winterbetrieb gewechselt ❄️
Heizkreis 2 hat in Sommerbetrieb gewechselt ☀️
Heizkreis 2 hat in Winterbetrieb gewechselt ❄️
Heizung hat in Frostmodus gewechselt ❄️
Heizkreis 2 hat in Frostmodus gewechselt ❄️
Heizung hat den Frostmodus verlassen ❄️
Heizkreis 2 hat den Frostmodus verlassen ❄️
```

Zudem jetzt noch die Bestätigung über einen empfangenen Befehl inkl. Wert:
<img width="531" alt="image" src="https://github.com/dewenni/ESP_Buderus_KM271/assets/46074831/2667661d-948e-4c90-9f04-0ae965770dff">

und eine Nachricht wenn Datum und Uhrzeit gesetzt werden, sowie zusätzliche Infos über den Restart-Grund
<img width="531" alt="image" src="https://github.com/dewenni/ESP_Buderus_KM271/assets/46074831/e93bbb5c-3b03-4b73-a2f8-c213c17a1c5f">

**Comment by dewenni** [comment]
implemented in [v4.0.4](https://github.com/dewenni/ESP_Buderus_KM271/releases/tag/v4.0.4)
